compiler/daml-lf-tools/src/DA/Daml/LF/TypeChecker/Check.hs

```diff
@@ -211,6 +212,14 @@ typeOfBuiltin = \case
   BEMapDelete -> pure $ TForall (alpha, KStar) $ TText :-> TMap tAlpha :-> TMap tAlpha
   BEMapToList -> pure $ TForall (alpha, KStar) $ TMap tAlpha :-> TList (TMapEntry tAlpha)
   BEMapSize   -> pure $ TForall (alpha, KStar) $ TMap tAlpha :-> TInt64
+  BEGenMapEmpty -> pure $ TForall (alpha, KStar) $ TForall (beta, KStar) $ TGenMap tAlpha tBeta
```

Comment:
Perhaps that is nitpicking but I would find this easier to read if we called the type variables `k` and `v` or `key` and `value`.